omod/src/main/webapp/admin/person/relationshipTypeViewForm.jsp

```diff
@@ -23,7 +23,7 @@
 		<c:forEach var="relationshipType" items="${relationshipTypeList}">
 			<tr>
 				<td><a href="relationshipType.form?relationshipTypeId=${relationshipType.relationshipTypeId}">
-					   ${relationshipType}
+					<c:out value="${relationshipType}"/>
```

Comment:
@Parth59  Have you tried testing this change on a running instance of openmrs?

Reply:
Hi Herbert. 
Apologies for the ticket issue. Have updated the link in the previous commit. 
I recently tried running it on https://demo.openmrs.org/openmrs/admin/person/relationshipTypeViews.form and the issue persists. So, Is it that the issue not reproducible at your system?

Reply:
> I recently tried running it on https://demo.openmrs.org/openmrs/admin/person/relationshipTypeViews.form and the issue persists. So, Is it that the issue not reproducible at your system?

Try to install a local copy of openmrs from which you can easily test your changes

Reply:
> > I recently tried running it on https://demo.openmrs.org/openmrs/admin/person/relationshipTypeViews.form and the issue persists. So, Is it that the issue not reproducible at your system?
> 
> Try to install a local copy of openmrs from which you can easily test your changes

Yup Yup. I actively test out on my local system.  It's just that since you mentioned if I have tested it out, I was unsure and that's why I verified in the demo version.